modules/lang-painless/src/main/resources/org/elasticsearch/painless/org.elasticsearch.script.score.txt

```diff
@@ -50,5 +50,7 @@ static_import {
     double cosineSimilarity(org.elasticsearch.script.ScoreScript, Object, String) bound_to org.elasticsearch.script.VectorScoreScriptUtils$CosineSimilarity
     double dotProduct(org.elasticsearch.script.ScoreScript, Object, String) bound_to org.elasticsearch.script.VectorScoreScriptUtils$DotProduct
     double hamming(org.elasticsearch.script.ScoreScript, Object, String) bound_to org.elasticsearch.script.VectorScoreScriptUtils$Hamming
+    double maxSimDotProduct(org.elasticsearch.script.ScoreScript, Object, String) bound_to org.elasticsearch.script.MultiVectorScoreScriptUtils$MaxSimDotProduct
```

Comment:
Why only dotProduct? Do we have plans to add cosine similarity later?

Reply:
@mayya-sharipova yeah, we can add it later. This PR was already hitting 1k+ LOC.